<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="powerLevelUpButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QToolButton" name="powerLevelDownButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>270</y>
      <width>26</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Up/Down Power Level (uA)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recordLastTreatmentButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Record Button</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="onOffButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>On/Off Button</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Treatment History</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>254</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="time20">
       <property name="text">
        <string>20 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="time40">
       <property name="text">
        <string>40 min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="time60">
       <property name="text">
        <string>60 min</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Treatment Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Waveform</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>254</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="alpha">
       <property name="text">
        <string>Alpha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="beta">
       <property name="text">
        <string>Beta</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gamma">
       <property name="text">
        <string>Gamma</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="historySelector">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="frequencyUI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Frequency (0.5, 77, or 100 Hz)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTreatment">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>START TREATMENT!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Treatment Button</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="displayHolder"/>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DISPLAY</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>360</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="electrodePadHolder"/>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>330</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ELECTRODE PAD</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
